From 3c6f74571b130fa368760d5566b4398e166a7a00 Mon Sep 17 00:00:00 2001
From: Chuck <fanck0605@qq.com>
Date: Wed, 15 Jul 2020 14:20:35 +0800
Subject: [PATCH] feeds: enable hell0world by default

Signed-off-by: Chuck <fanck0605@qq.com>
---
 feeds.conf.default | 1 +
 1 file changed, 1 insertion(+)

diff --git a/feeds.conf.default b/feeds.conf.default
index 416da4b0c..e0474e71d 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -6,3 +6,4 @@ src-git telephony https://git.openwrt.org/feed/telephony.git
 #src-git targets https://github.com/openwrt/targets.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
+src-git helloworld https://github.com/fw876/helloworld
--
2.30.1

