From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chuck <fanck0605@qq.com>
Date: Tue, 21 Jul 2020 13:13:10 +0800
Subject: [PATCH] version.mk: use tencent's opkg mirror

Signed-off-by: Chuck <fanck0605@qq.com>
---
 include/version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/version.mk b/include/version.mk
index b7f42e13bb..b558563db8 100644
--- a/include/version.mk
+++ b/include/version.mk
@@ -32,7 +32,7 @@ VERSION_CODE:=$(call qstrip,$(CONFIG_VERSION_CODE))
 VERSION_CODE:=$(if $(VERSION_CODE),$(VERSION_CODE),$(REVISION))
 
 VERSION_REPO:=$(call qstrip,$(CONFIG_VERSION_REPO))
-VERSION_REPO:=$(if $(VERSION_REPO),$(VERSION_REPO),https://downloads.openwrt.org/snapshots)
+VERSION_REPO:=$(if $(VERSION_REPO),$(VERSION_REPO),https://mirrors.cloud.tencent.com/openwrt/snapshots)
 
 VERSION_DIST:=$(call qstrip,$(CONFIG_VERSION_DIST))
 VERSION_DIST:=$(if $(VERSION_DIST),$(VERSION_DIST),OpenWrt)
-- 
2.17.1

