From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chuck <fanck0605@qq.com>
Date: Sat, 20 Feb 2021 17:09:49 +0800
Subject: [PATCH] openssl: optimize for speed instead of size
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

对于大多数软路由设备来说，牺牲一点存储空间来换取性能是值得的。即使带来的提升可能并不大，但是大多数设备也并不缺那点空间。

Signed-off-by: Chuck <fanck0605@qq.com>
---
 package/libs/openssl/Config.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/package/libs/openssl/Config.in
+++ b/package/libs/openssl/Config.in
@@ -4,7 +4,7 @@ comment "Build Options"
 
 config OPENSSL_OPTIMIZE_SPEED
 	bool
-	default y if x86_64 || i386
+	default y if !SMALL_FLASH
 	prompt "Enable optimization for speed instead of size"
 	select OPENSSL_WITH_ASM
 	help
@@ -16,7 +16,7 @@ config OPENSSL_OPTIMIZE_SPEED
 
 config OPENSSL_WITH_ASM
 	bool
-	default y if !SMALL_FLASH || !arm
+	default y if !SMALL_FLASH
 	prompt "Compile with optimized assembly code"
 	depends on !arc
 	help
