--- a/package/custom/luci-app-openclash/root/etc/config/openclash
+++ b/package/custom/luci-app-openclash/root/etc/config/openclash
@@ -29,7 +29,7 @@ config openclash 'config'
 	option log_level 'silent'
 	option proxy_mode 'rule'
 	option intranet_allowed '1'
-	option enable_udp_proxy '1'
+	option enable_udp_proxy '0'
 	option disable_udp_quic '1'
 	option lan_ac_mode '0'
 	option operation_mode 'redir-host'
