--- a/package/custom/luci-app-openclash/root/etc/config/openclash
+++ b/package/custom/luci-app-openclash/root/etc/config/openclash
@@ -39,7 +39,7 @@ config openclash 'config'
 	option filter_aaaa_dns '0'
 	option small_flash_memory '0'
 	option interface_name '0'
-	option common_ports '0'
+	option common_ports '1'
 	option log_size '1024'
 	option tolerance '0'
 	option store_fakeip '1'
--- a/package/custom/luci-app-openclash/luasrc/model/cbi/openclash/settings.lua
+++ b/package/custom/luci-app-openclash/luasrc/model/cbi/openclash/settings.lua
@@ -113,7 +113,7 @@ o.default = 0
 
 o = s:taboption("op_mode", Flag, "common_ports", font_red..bold_on..translate("Common Ports Proxy Mode")..bold_off..font_off)
 o.description = translate("Only Common Ports, Prevent BT/P2P Passing")
-o.default = 0
+o.default = 1
 o:depends("en_mode", "redir-host")
 o:depends("en_mode", "redir-host-tun")
 o:depends("en_mode", "redir-host-mix")
