--- a/applications/luci-app-smartdns/htdocs/luci-static/resources/view/smartdns/smartdns.js
+++ b/applications/luci-app-smartdns/htdocs/luci-static/resources/view/smartdns/smartdns.js
@@ -234,6 +234,12 @@ return view.extend({
 			_("Maximum TTL for all domain result."));
 		o.rempty = true;
 
+		// bind arguments;
+		o = s.taboption("settings", form.Value, "bind_args", _("Bind arguments"),
+			_("Specify bind arguments."));
+		o.rmempty = true;
+		o.rempty = true;
+
 		// Eanble;
 		o = s.taboption("seconddns", form.Flag, "seconddns_enabled", _("Enable"),
 			_("Enable or disable second DNS server."));
