--- a/package/openwrt/files/etc/init.d/smartdns
+++ b/package/openwrt/files/etc/init.d/smartdns
@@ -275,17 +275,19 @@ load_service()
 	config_get port "$section" "port" "6053"
 	config_get ipv6_server "$section" "ipv6_server" "1"
 	config_get tcp_server "$section" "tcp_server" "1"
+	config_get bind_args "$section" "bind_args" ""
+
 	if [ "$ipv6_server" = "1" ]; then
-		conf_append "bind" "[::]:$port"
+		conf_append "bind" "[::]:$port $bind_args"
 	else
-		conf_append "bind" ":$port"
+		conf_append "bind" ":$port $bind_args"
 	fi
 
 	[ "$tcp_server" = "1" ] && {
 		if [ "$ipv6_server" = "1" ]; then
-			conf_append "bind-tcp" "[::]:$port"
+			conf_append "bind-tcp" "[::]:$port $bind_args"
 		else
-			conf_append "bind-tcp" ":$port"
+			conf_append "bind-tcp" ":$port $bind_args"
 		fi
 	}
 	config_get dualstack_ip_selection "$section" "dualstack_ip_selection" "0"
