--- a/package/openwrt/files/etc/init.d/smartdns
+++ b/package/openwrt/files/etc/init.d/smartdns
@@ -386,6 +386,10 @@ start_service()
 {
 	config_load "smartdns"
 	config_foreach load_service "smartdns"
+	grep smartdns/update_chinalist.sh /etc/crontabs/root >/dev/null || {
+		echo '0 3 * * * /usr/lib/smartdns/update_chinalist.sh' >>/etc/crontabs/root
+		/etc/init.d/cron reload
+	}
 }
 
 reload_service()
