--- a/package/openwrt/custom.conf
+++ b/package/openwrt/custom.conf
@@ -11,4 +11,10 @@
 # log-num 2
 
 # List of hosts that supply bogus NX domain results 
-# bogus-nxdomain [ip/subnet]
\ No newline at end of file
+# bogus-nxdomain [ip/subnet]
+
+conf-file /etc/smartdns/nameserver-base.china.conf
+conf-file /etc/smartdns/nameserver-google.china.conf
+conf-file /etc/smartdns/nameserver-apple.china.conf
+
+conf-file /etc/smartdns/bogus-nxdomain.china.conf
--- a/package/openwrt/files/etc/config/smartdns
+++ b/package/openwrt/files/etc/config/smartdns
@@ -1,5 +1,6 @@
 config 'smartdns'
 	option 'enabled' '1'
+	option server_group 'oversea'
 
 config server
 	option type 'udp'
