--- a/package/openwrt/address.conf
+++ b/package/openwrt/address.conf
@@ -5,6 +5,13 @@
 # address /www.example.com/-, ignore address, query from upstream, suffix 4, for ipv4, 6 for ipv6, none for all
 # address /www.example.com/#, return SOA to client, suffix 4, for ipv4, 6 for ipv6, none for all
 
+address /doh.pub/1.12.12.12
+address /dns.alidns.com/223.5.5.5
+address /cloudflare-dns.com/1.1.1.1
+address /dns.google/8.8.8.8
+address /doh.opendns.com/208.67.222.222
+address /dns.quad9.net/9.9.9.9
+
 # specific ipset to domain
 # ipset /domain/[ipset|-]
 # ipset /www.example.com/block, set ipset with ipset name of block 
